docs/includes/generated_docs/schemas/beta.raw.tpp.md

```diff
@@ -172,14 +172,25 @@ This table contains some historical APCS data.
 <p class="dimension-indicator"><code>many rows per patient</code></p>
 ## isaric
 
-A subset of the ISARIC data.
+A subset of the ISARIC data,
+sourced from the [ISARIC COVID-19 Clinical Database][isaric_clinical_database].
 
 These columns are deliberately all taken as strings while in a preliminary phase.
```

Comment:
All columns included have deliberately been taken as strings while in a preliminary phase.

Reply:
Edited to the suggestion here.